### PR TITLE
Customisations and few fixes for online DQM

### PR DESCRIPTION
**Pull request by dmitrijus**
this deprecates https://github.com/cms-sw/cmssw/pull/10563

Small set of fixes for online DQM:
- fix the name of the online playback GUI host
- new online GT for DQM
- fix temporary connection string for Tau Tag high-level calibrations (ping to @diguida - oh, it's me! - and @mmusich )

### CODE REVIEW

**Comment by deguio** [comment]
+1
@davidlange6 please merge before the next 74 build.
online only.

**Comment by deguio** [comment]
please test

**Comment by slava77** [comment]
@cmsbuild please test
... unclear why it failed to merge otherwise

**Comment by deguio** [comment]
please test
let me try again...
F.

**Comment by dmitrijus** [comment]
I've rebased this on top of 7_4_X, please test once more.

**Comment by diguida** [comment]
please test

**Comment by danduggan** [comment]
+1
for online only

**Comment by cvuosalo** [comment]
@cmsbuild please test
Comparisons got stuck in Queued state. Try test again.

**Comment by cvuosalo** [comment]
@cmsbuild please
DAS may be working

**Comment by cvuosalo** [comment]
@cmsbuild please test
Try DAS yet again
